server/webhook.go

```diff
@@ -648,9 +656,11 @@ func (s *Server) processFilesInProject(ctx context.Context, pushEvent vcs.PushEv
 				migrationDetailListForFile, activityCreateListForFile := s.prepareIssueFromSDLFile(ctx, repo, pushEvent, fileInfo.migrationInfo, fileInfo.item.FileName)
 				activityCreateList = append(activityCreateList, activityCreateListForFile...)
 				if len(migrationDetailListForFile) != 0 {
-					issueName := fmt.Sprintf("Apply schema diff by file %s", strings.TrimPrefix(fileInfo.item.FileName, repo.BaseDirectory+"/"))
-					creatorID := s.getIssueCreatorID(ctx, pushEvent.FileCommit.AuthorEmail)
-					if err := s.createIssueFromMigrationDetailList(ctx, issueName, pushEvent, creatorID, repo.ProjectID, migrationDetailListForFile); err != nil {
+					databaseName := fileInfo.migrationInfo.Database
+					issueName := fmt.Sprintf(issueNameTemplate, databaseName, "Alter schema")
+					issueDescription := fmt.Sprintf("Apply schema diff by file %s", strings.TrimPrefix(fileInfo.item.FileName, repo.BaseDirectory+"/"))
+					creatorID := s.getIssueCreatorID(ctx, pushEvent.CommitList[0].AuthorEmail)
```

Comment:
creatorID Should be calculated in the very beginning instead of here in every iteration.

Reply:
Done.